coord/src/rmr/command.h

```diff
@@ -37,6 +39,9 @@ typedef struct {
   /* Whether the command chain is depleted - don't resend */
   bool depleted;
 
+  // Root command for current response
+  MRRootCommand rootCommand;
```

Comment:
Does this addition change the size of the structure? If so, consider making `forCursor` and `depleted` flags and check if we need to limit the size of this member (if the compiler assigns it more than 8 bits).

Reply:
It does change the size of the structure, but this is unavoidable, as the previous size is 5 bytes exactly - such that we have no room to play with.